spec/lib/driver_commission_history_response_spec.rb

```diff
@@ -6,10 +6,22 @@
 require 'support/vcr'
 
 RSpec.describe GmanClient do
+  let(:url) { 'http://localhost:3000' }
+  let(:token_url) { 'http://localhost:3000/oauth/token' }
+  # rubocop:disable Metrics/LineLength
+  let(:client_id) { 'd137a94543da86e52214d3ed86b015f9299ad2fc66681637600afad814cd7d2b' }
```

Comment:
If you look at 
https://github.com/westernmilling/gman_client/blob/master/spec/vcr_cassettes/driver_commission_histories.yml#L8
You can see that the grant type uses client_id and client_secret.